lib/Language/Souffle/Class.hs

```diff
@@ -64,19 +64,19 @@ type family IsInput fact dir where
   IsInput _ 'Input = ()
   IsInput _ 'InputOutput = ()
   IsInput fact dir = TypeError
-    ( 'Text "You tried to use an " ':<>: 'ShowType (FormatDirection dir) ':<>: 'Text " fact of type " ':<>: 'ShowType fact ':<>: 'Text " as an input."
-    ':$$: 'Text "Possible solution: change the FactDirection of " ':<>: 'ShowType fact
-      ':<>: 'Text " to either 'Input' or 'InputOutput'."
+    ( 'Text "You tried to use an " ' :<>: 'ShowType (FormatDirection dir) ' :<>: 'Text " fact of type " ' :<>: 'ShowType fact ' :<>: 'Text " as an input."
```

Comment:
Are these spaces required now between the tick and the symbol name? Preferably  I want them like before (together).